### DISCUSSION TITLE
webdav上传总是不正常，真是奇怪了。

### DISCUSSION THREAD

**Discussion by fzlychee2019** [AlistGo/alist#1210] [General]
用clouddrive挂载的天翼、阿里云盘等使用rclone上传都特别稳定，但是用ALIST挂载的WEBDAV使用rclone上传却总是各种出错 。从来无法正常使用。上传几个文件可能感觉不到，上传一大批文件时，就会看到各种出错了。

**Comment by githubgj** [comment]
我挂载的是移动139网盘，用rclone上传文件，同样是小文件可以上传，大文件上传没多久就报405 method not allowded，请问大家后来解决了吗？
